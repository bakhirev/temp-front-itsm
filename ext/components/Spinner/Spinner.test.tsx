import React from 'react';
import { shallow } from 'enzyme';

import { Spinner } from './';

describe('Spinner', () => {
  it('should render component with big size by default', () => {
    const wrapper = shallow(<Spinner />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render component with medium size', () => {
    const wrapper = shallow(<Spinner size="medium" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render component with small size', () => {
    const wrapper = shallow(<Spinner size="small" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render component with inverse color', () => {
    const wrapper = shallow(<Spinner inverse />);
    expect(wrapper).toMatchSnapshot();
  });
});
