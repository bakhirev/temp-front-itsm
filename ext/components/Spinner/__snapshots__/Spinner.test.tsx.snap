// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render component with big size by default 1`] = `
<styled.div
  size="big"
>
  <div />
</styled.div>
`;

exports[`Spinner should render component with inverse color 1`] = `
<styled.div
  size="big"
>
  <div />
</styled.div>
`;

exports[`Spinner should render component with medium size 1`] = `
<styled.div
  size="medium"
>
  <div />
</styled.div>
`;

exports[`Spinner should render component with small size 1`] = `
<styled.div
  size="small"
>
  <div />
</styled.div>
`;
